packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -144,7 +144,7 @@ Did you forget to run "shorebird init"?''',
       return ExitCode.software.code;
     }
 
-    final platform = results['platform'] as String;
+    const platform = 'android';
```

Comment:
Thanks for catching and fixing this!

Reply:
The `shorebird release` command is also not working as-is - it seems like commands with subcommands "branch" commands and can't be run themselves (see https://pub.dev/documentation/args/latest/command_runner/Command-class.html). Seems like we may need to deprecate `shorebird release` sooner than we thought 🙃

Reply:
Hmm I thought it was still possible 🤔

Will look into it in a few mins